docs/en/authenticators.rst

```diff
@@ -477,19 +484,26 @@ authenticators must send specific challenge headers in the response::
     // Instantiate the service
     $service = new AuthenticationService();
 
-    // Load identifiers
-    $service->loadIdentifier('Authentication.Password', [
-        'fields' => [
-            'username' => 'email',
-            'password' => 'password'
-        ]
-    ]);
-    $service->loadIdentifier('Authentication.Token');
+    // Define identifiers
+    $passwordIdentifier = [
+        'Authentication.Password', [
```

Comment:
```suggestion
        'Authentication.Password' => [
```

Is the identifer config a Map, or a tuple? I didn't see a test covering either form of config.

Reply:
Assoc
You are right